src/azure-cli/requirements.py3.windows.txt

```diff
@@ -63,7 +63,7 @@ azure-mgmt-monitor==3.0.0
 azure-mgmt-msi==6.0.0
 azure-mgmt-netapp==7.0.0
 azure-mgmt-network==19.3.0
-azure-mgmt-policyinsights==1.0.0
+azure-mgmt-policyinsights==1.1.0b2
```

Comment:
Why the SDK package is a beta version?

Reply:
this is the latest python sdk version https://pypi.org/project/azure-mgmt-policyinsights/1.1.0b2/

Reply:
In fact, what I want to ask is why the latest SDK version is a beta version, is because it contains some unstable features or big changes?
If so, will it affect the existing commands?

Reply:
this fixes an [issue ](https://github.com/Azure/azure-cli/issues/22173) we had ran into wherein the python sdk was making GET calls for nextLink instead of POST calls. Fix was needed in [Policy Insights swagger](https://github.com/Azure/azure-rest-api-specs/pull/14035) and 
[py sdk release](https://github.com/Azure/sdk-release-request/issues/1861)

in this PR bumping it to 1.1.0b2 fixes the issue in cli